views/download.handlebars

```diff
@@ -6,64 +6,67 @@
     {{> sentry dsn=dsn}}
   {{/if}}
   <script src="/bundle.js"></script>
-  <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css" />
   <link rel="stylesheet" type="text/css" href="/main.css" />
   {{#if trackerId}}
     {{> analytics trackerId=trackerId}}
   {{/if}}
 </head>
 <body>
 
-  <div class="main-window">
-    <div id="download">
-      {{#if filename}}
+  <div id="download">
+    {{#if filename}}
+    <div id="download-page-one">
       <div class="title">
-        Your friend is sending you a file: <br />
-        {{{filename}}} ({{{filesize}}})
+        Download {{{filename}}} ({{{filesize}}})
       </div>
-      <div id="download-page-one">
-        <div>
-          <button id="download-btn">Download File</button>
-        </div>
-        <div id='expired-img'>
-          <img src='/resources/link_expired.png' />
-        </div>
+      <div class="description">
+        Your friend is sending you a file with Firefox Send, a service that allows you to share files with a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
       </div>
+      <img src="/resources/illustration_download.svg" id="download-img"/>
+      <div>
+        <button id="download-btn" title="Download">Download</button>
+      </div>
+    </div>
 
-
-      <div id="download-progress">
-        <div  id="download-text">
-          Downloading File...
-        </div>
-        <div class="upload">
-          <!-- progress bar here -->
-          <div id="progress-bar"></div>
-          <div id="progress-text"></div>
+    <div id="download-progress">
+      <div class="title">
+        Downloading {{{filename}}} ({{{filesize}}})
+      </div>
+      <div class="description">
+        Please leave this tab open while we fetch your file and decrypt it.
+      </div>
+      <!-- progress bar here -->
+      <div class="progress-bar" id="dl-progress">
+        <div class="percentage">
+          <span class="percent-number"></span>
+          <span class="percent-sign">%</span>
```

Comment:
Personal curiosity, but couldn't we just do this w/ CSS and save a couple nested `<span>`s?

```html
<div class="percentage">13</div>
```
```css
.percentage::after {
  content: "%";
}
```